editions/tw5.com/tiddlers/demonstrations/SampleWizard.tid

```diff
@@ -1,6 +1,7 @@
 created: 20140912145537860
 footer: <$button message="tm-close-tiddler">Close</$button>
-modified: 20140912145537861
+modified: 20210621223203244
+prevent-scroll-chain: yes
```

Comment:
I would recommend using a field name that is more intuitive for end users, e.g. `capture-scrolling`

Reply:
"capture" is not related to the underlying `overscroll-behavior: contain` at all, maybe `contain-scroll`?

Not gonna bike-shed this though, go with what fits best, just please keep in mind that whenever you invent new terminology to help users who don't know the standards, you also force those who do to remember twice as much: how the standard calls it plus the new end-user friendly terms, so it's always a tradeoff.

Reply:
The stated term was an example of an alternative and not a firm proposal. Note however that the implementation in this PR does not use `overscroll-behaviour`.

User demographic matters a great deal in such determinations and the vast majority of TW users are decidedly non-technical.

Reply:
>The stated term was an example of an alternative and not a firm proposal. Note however that the implementation in this PR does not use overscroll-behaviour

Yea, .. It doesn't work reliably. ... The parameter is only active if there is something to scroll. .. So only if the content of the modal is bigger than the y-size of the modal. Since many modals have less content it doesn't work there.

Reply:
The field-name used in the PR isn't carved in stone. I would be OK with the field-name: `contain-scroll` ... I did use a name related to the technical description at MDN.

@Jermolene what do you think?

>from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior) 2nd paragraph
>
>You may also have noticed that when you have a dialog box with scrolling content on top of a page of scrolling content, once the dialog box's scroll boundary is reached, the underlying page will then start to scroll — this is called **scroll chaining**.